src/Driver/Driver.jl

```diff
@@ -121,6 +121,19 @@ function get_setting(setting_name::Symbol, settings, defaults)
     end
 end
 
+function get_gpu_setting(setting_name::Symbol, settings, defaults)
+    # do not override disable_gpu kwargs setting if it exists
+    if !haskey(settings, setting_name)
+        # if old GPU ENV keyword exists, overwrite the settings variable if not defined
+        if haskey(ENV, "CLIMATEMACHINE_GPU")
```

Comment:
it won't show the warning in the CLI.  I believe that this would be annoying for users due to where it has to be checked (it's different than importing `ClimateMachine.init()` and the warning showing up inline).  We don't throw a warning for the other deprecated ENV variables in the cli